psiminer-core/src/test/kotlin/fomatter/JsonTreeFormatterTest.kt

```diff
@@ -0,0 +1,155 @@
+package fomatter
+
+import BasePsiRequiredTest
+import com.intellij.openapi.application.ReadAction
+import formatter.tree.JsonTreeFormatter
+import labelextractor.LabeledTree
+import org.junit.Assert
+import org.junit.jupiter.api.Test
+import psi.transformations.typeresolve.resolvedTokenType
+
+internal class JsonTreeFormatterTest : BasePsiRequiredTest() {
+
+    @Test
+    fun collectNodeRepresentationTest() = ReadAction.run<Exception> {
+        val psiRoot = getJavaMethod(emptyMethod)
+        psiRoot.resolvedTokenType = "type"
```

Comment:
It seems that you don't need to add types into PSI for this test, because its purpose is to check the correct JSON, not the type resolving.